dist/index.js

```diff
@@ -121123,6 +121122,12 @@ function run() {
                 artifacts = artifacts.filter(artifact => matcher.match(artifact.name));
                 core.debug(`Filtered from ${listArtifactResponse.artifacts.length} to ${artifacts.length} artifacts`);
             }
+            else {
+                core.info('No input name or pattern filtered specified, downloading all artifacts');
+                if (!inputs.mergeMultiple) {
```

Comment:
This option is also valid if a pattern is specified, right?  It'd be nice to have the "extra directory" message in that case too.